src/components/widget/CustomNumberInput.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+const FLOAT_NUMBER_REGEX_FORMAT = /^-?[0-9]+[.,]?[0-9]*$/;
```

Comment:
I think people will want to write e.g. `,1` for `0.1`...
maybe...if it's easy...and you now even have a test to cover your back...
could you look into that pls 😊 ?

Reply:
We've never supportad that but I might as well just add that.